### PR TITLE
4.3 fixes

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/catalog.xyaml [hunk 1]
```diff
@@ -150,4 +150,19 @@ browse_to_install_plan:
       xpath: //a[contains(@href, 'InstallPlan')]
     timeout: 60
     op: click
+click_project_list_dropdown:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/catalog.xyaml]: rules added for `select_target_namespace`

File: lib/rules/web/admin_console/4.3/yaml.xyaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+open_view_sidebar:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/yaml.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 3]
```diff
@@ -88,11 +88,24 @@ click_overview_tab:
   params:
     tab_name: Overview
   action: click_tab
+click_yaml_tab:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 4]
```diff
@@ -88,11 +88,24 @@ click_overview_tab:
...
+  params:
+    tab_name: YAML
+  action: click_tab
 click_radio_input:
   element:
     selector:
       xpath: //input[@type='radio' and @value='<radio_text>']
     op: click
+click_close_icon:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 5]
```diff
@@ -88,11 +88,24 @@ click_overview_tab:
...
+  element:
+    selector:
+      xpath: //button[contains(@aria-label,'Close')]
+    op: click
+click_yaml_create_button:
+  params:
+    button_id: yaml-create
+  action: click_button_by_id
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 6]
```diff
@@ -88,11 +88,24 @@ click_overview_tab:
   params:
     tab_name: Overview
   action: click_tab
+click_yaml_tab:
+  params:
+    tab_name: YAML
+  action: click_tab
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.3/yaml.xyaml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  element:
+    selector:
+      xpath: //li[contains(@class,'co-resource-sidebar-item')]/h5[contains(.,'<field>')]/following-sibling::button[contains(.,'View details')]
+    op: click
+click_item_in_breadcrumb:
+  element:
+    selector:
+      xpath: //li[contains(@class,'pf-c-breadcrumb__item')]//button[contains(.,'<breadcrumb_item>')]
+    op: click
+
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/yaml.xyaml]: rules copied for OCP-23967 to run on 4.3

File: lib/rules/web/admin_console/4.4/catalog.xyaml [hunk 8]
```diff
@@ -150,4 +150,19 @@ browse_to_install_plan:
...
+  element:
+    selector:
+      xpath: //ul//a[@id='<project_name>-Project-link' and contains(@class,'dropdown__menu-item')]
+    op: click
+select_target_namespace:
+  params:
+    install_mode: OwnNamespace
+  action: choose_installmode
+  action: click_project_list_dropdown
+  action: choose_project_from_dropdown_list
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/catalog.xyaml]: rules added for `select_target_namespace`

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 9]
```diff
@@ -60,7 +60,7 @@ select_resource_item:
 click_on_resource_name:
   element:
     selector:
-      xpath: //a[contains(@class,'co-resource-item__resource-name') and text()='<item>']
+      xpath: //a[contains(@class,'co-resource-item__resource-name') and text()='<item>'] | //span[contains(@class,'co-resource-item__resource-name') and text()='<item>']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: xpath changes

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml (hunk 6 above)
Comment (SSshahan) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Thanks！

**Comment by yapei** [comment]
@pruan-rht  @liangxia  Can you help merge? Thanks

**Comment by yapei** [comment]
@liangxia  @pruan-rht  Can you help merge it? Thanks

**Pull request merged**